code-challenges.test.js

```diff
@@ -49,4 +121,32 @@ const hand3 = [5, 5, 5, 5, 4]
 const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
+expect(fullHouse(hand1)).toEqual(true)
+expect(fullHouse(hand2)).toEqual(false)
+expect(fullHouse(hand3)).toEqual(false)
+})
+})
+
+// ReferenceError: fullHouse is not defined
+
 // b) Create the function that makes the test pass.
+
+// PsuedoCode;
+// Declare a function called fullHouse
+// Set the parameter array
+// Create a for loop to interate through each value in the array
+// Utilize conditionals to determine if the array is a full house
+// if [0] === [1] ===[2] && [3] === [4], return true
+// else return false 
+
+
+const fullHouse = (array) => {
+  for (let i=0; i < array.length; i++)
+  if ((array[0] === array[1]) && (array[1] === array[2]) && (array[3] === array[4])) {
+return true
+  } else {
+return false 
+}
+ }
```

Comment:
This is excellent problem solving! push yourself to express your pseudo code in plain language once it's solved even. 
ex .
- if value at the zeroth index matches the value of the array at the index one and ...